kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/ProcurementCardHolder.xml

```diff
@@ -205,6 +205,9 @@
 		<property name="label" value="Card Holder Postal (ZIP) Code" />
 		<property name="shortLabel" value="Code" />
 		<property name="required" value="false"/>
+		<property name="validationPattern">
+			<bean parent="AnyCharacterValidationPattern"/>
+		</property>

```

Comment:
There is a maxLength in the parent's parent (PostalCode-parentBean), set to size 20, which should be sufficient for any incoming data (the spec says that there isn't supposed to be any validation on this field). The original mod had no change to the maxLength either.